presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -76,6 +76,7 @@
     private boolean pushAggregationThroughJoin = true;
     private double memoryRevokingTarget = 0.5;
     private double memoryRevokingThreshold = 0.9;
+    private boolean parseDecimalLiteralsAsDouble;
```

Comment:
@martint , to facilitate a transition period, should this be  `true` for some time?

Reply:
Yes, let's keep it as `true` for a little bit. It's a very invasive change (in terms of behavior), so I want to keep surprises to a minimum.

Reply:
Hm, do you have preference how this should be handled in tests? Most (if not all) of the changes in "Fix failing tests in presto-main", "Fix failing tests in other modules" would need to be backed-off if `parseDecimalLiteralsAsDouble` defaults to `true`. Alternatively those tests would explicitly ask to use session with this flagged swapped..
@martint, what do you think? Any other options?

Reply:
There's no good answer to this. I'd make the tests explicitly ask to turn on this setting (for now), and introduce some basic tests to make sure we don't break anything when the legacy setting is on.